projects/components/src/lib/directives/aria-activedescendant/aria-activedescendant.directive.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+import { IdGenerator } from '../../../utils';
+import { AriaActivedescendantService } from './aria-activedescendant.service';
```

Comment:
Please name it capitalizing all the words, `AriaActiveDescendantService`, `vcdAriaActiveDescendant`

Reply:
Also, please document public classes/methods/properties